File: db/dbconn.php
<?php 

print ("Connecting to MySQL...."); 

try { 

$pdo = new PDO('mysql:host=localhost;dbname=dbtest;charset=utf8','takano','');


} catch (PDOException $e) { 

    exit('DB connection error.'.$e->getMessage()); 

} 

print ("Exit."); 

?> 

File: db/dbconn.py
import mysql.connector

print ("Connecting to MySQL....");

try:
  cnx = mysql.connector.connect(user='takano', password='',
                              host='localhost',
                              database='dbtest')
except mysql.connector.Error as err:
  print("DB connection error {}".format(err) )

print("Exit.")

cnx.close()
